index.html

```diff
@@ -65,7 +65,14 @@
             <input id="arm-position" type="range" min="0" max="1200" value="770">
         </div>
     </aside>
-    <script src="/networktables/networktables.js"></script>
+    <div class="greybox">
```

Comment:
When the popup shows itself two things happen:
- The background is slightly grayed out
- Anything behind the box graying out the background is blured